README.md

```diff
@@ -0,0 +1,111 @@
+# Roundcube
+
+Sets up and configures a roundcube instance.
+
+
+## Requirements
+
+It needs an apt based system like Ubuntu. Also the [stuvusIT.nginx](https://github.com/stuvusIT/nginx), [stuvusIT.php-fpm](https://github.com/stuvusIT/php-fpm) and [stuvusIT.mariadb](https://github.com/stuvusIT/mariadb) roles are required.
+
+
+## Role Variables
+
+| Name                              | Required                 | Default      | Description                                                                                                                                                                                            |
+|:----------------------------------|:------------------------:|:-------------|:-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `global_cache_dir`                | :heavy_check_mark:       |              | Cache directory to download Mattermost files to                                                                                                                                                        |
+| `roundcube_sql_password`              | :heavy_check_mark: |       | Password of the `roundcube_sql_user`                                                                                                                                                                          |
+| `roundcube_des_key`              | :heavy_check_mark: |  | This key is used to encrypt the users imap password which is stored in the session record (and the client cookie if remember password is enabled). Please provide a string of exactly 24 chars.                                                                                                                                                                                |
```

Comment:
please also ensure that the key has this length

Reply:
done